src/components/Header/Header.tsx

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import { TodoUpdateContext, TodosContext } from '../../context/TodosContext';
+import { Todo } from '../../types/Todo';
+import { USER_ID } from '../../constants/USER_ID';
+import { Errors } from '../../types/Errors';
+
+export const Header: React.FC = () => {
+  const [titleField, setTitleField] = useState('');
+  const [isAddingTodo, setIsAddingTodo] = useState(false);
+  const [allCompleted, setAllCompleted] = useState(false);
```

Comment:
if you have a boolean value than start a name with `is` or `has` like you did with `isAddingTodo`

please update `allCompleted`